src/leihs/my/back/html.clj

```diff
@@ -1,25 +1,22 @@
 (ns leihs.my.back.html
-  (:refer-clojure :exclude [str keyword])
+  (:refer-clojure :exclude [keyword str])
   (:require
-   [clojure.java.jdbc :as jdbc]
-   [hiccup.page :refer [html5 include-js]]
-   [leihs.core.json :refer [to-json]]
-   [leihs.core.shared :refer [head]]
-   [leihs.core.sql :as sql]
-   [leihs.core.http-cache-buster2 :as cache-buster]
-   [leihs.core.url.core :as url]
-   [leihs.my.authorization :as auth]
-   [leihs.core.remote-navbar.shared :refer [navbar-props]]))
+    [hiccup.page :refer [html5 include-js]]
+    [honey.sql :refer [format] :rename {format sql-format}]
+    [leihs.core.http-cache-buster2 :as cache-buster]
+    [leihs.core.json :refer [to-json]]
+    [leihs.core.remote-navbar.shared :refer [navbar-props]]
+    [leihs.core.shared :refer [head]]
+    [leihs.core.url.core :as url]
+    [leihs.my.authorization :as auth]
+    [next.jdbc :as jdbc]))
 
 (defn route-user [request]
   (let [user-id (-> request :route-params :user-id)
-        tx (:tx request)]
-    (-> (sql/select :*)
-        (sql/from :users)
-        (sql/where [:= :id user-id])
-        sql/format
-        (->> (jdbc/query tx))
-        first)))
+        tx (:tx-next request)
+        query (sql-format {:select [:*] :from [:users] :where [:= :id [:cast user-id :uuid]]})]
```

Comment:
prefer the honeysql functions representation anyway, otherwise each keyword on a newline.